### ISSUE TITLE
NTR: AGAT low in lymphocytes (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8142]
New term request AGAT low in lymphocytes (segal_080123183328): Arginine:glycine amidinotransferase

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000572
- label: Reduced tissue arginine:glycine amidinotransferase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:22386973
- def: Activity of L-arginine:glycine amidinotransferase (GATM; EC 2.1.4.1) in the tissues below the lower limit of normal. GATM activity can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8142
- synonyms:
